sdk/keyvault/keyvault-common/package.json

```diff
@@ -48,8 +48,15 @@
     "tslib": "^2.0.0"
   },
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.0.0",
+    "@typescript-eslint/parser": "^2.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "eslint": "^6.1.0",
+    "eslint-config-prettier": "^6.0.0",
+    "eslint-plugin-no-null": "^1.0.2",
+    "eslint-plugin-no-only-tests": "^2.3.0",
+    "eslint-plugin-promise": "^4.1.1",
```

Comment:
this PR deletes those dependencies from across our packages https://github.com/Azure/azure-sdk-for-js/pull/11528. Is there a specific reason for adding those back?

Reply:
I'll check! I think my master branch could've been outdated.

Reply:
Done! Please review again! 🌞